pom.xml

```diff
@@ -126,22 +126,38 @@
         <artifactId>variant</artifactId>
         <version>1.1</version>
       </dependency>
+      <dependency>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>github-branch-source</artifactId>
+        <version>2.7.1</version>

```

Comment:
Really?  Have you considered updating to version that isn't almost a year old?  There's been a ton of work in this area in that past year.

Reply:
Upgrading to the latest introduces a lot of enforcer issues, but i'll give it a go.

Reply:
I'll try and introduce the plugin bom in a separate PR and update some of the dependencies